egs/chime4/s5_1ch_track/local/clean_wsj0_data_prep.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+set -e
+
+# Copyright 2009-2012  Microsoft Corporation  Johns Hopkins University (Author: Daniel Povey)
+# Apache 2.0.
+
+# This is modified from the script in standard Kaldi recipe to account
+# for the way the WSJ data is structured on the Edinburgh systems.
+# - Arnab Ghoshal, 29/05/12
+
+# Modified from the script for CHiME2 baseline
+# Shinji Watanabe 02/13/2015
+
+if [ $# -ne 1 ]; then
+  printf "\nUSAGE: %s <original WSJ0 corpus-directory>\n\n" `basename $0`
+  echo "The argument should be a the top-level WSJ corpus directory."
+  echo "It is assumed that there will be a 'wsj0' and a 'wsj1' subdirectory"
+  echo "within the top-level corpus directory."
+  exit 1;
+fi
+
+wsj0=$1
+
+dir=`pwd`/data/local/data
```

Comment:
I think he just feels it looks better, and it's certainly more efficient
(no unnecessary command invocation).  Not a big deal though.


On Mon, Apr 25, 2016 at 1:44 PM, Shinji Watanabe <notifications@github.com>
wrote:

> In egs/chime4/s5_1ch_track/local/clean_wsj0_data_prep.sh
> <https://github.com/kaldi-asr/kaldi/pull/730#discussion_r60955587>:
>
> > +# - Arnab Ghoshal, 29/05/12
> > +
> > +# Modified from the script for CHiME2 baseline
> > +# Shinji Watanabe 02/13/2015
> > +
> > +if [ $# -ne 1 ]; then
> > +  printf "\nUSAGE: %s <original WSJ0 corpus-directory>\n\n" `basename $0`
> > +  echo "The argument should be a the top-level WSJ corpus directory."
> > +  echo "It is assumed that there will be a 'wsj0' and a 'wsj1' subdirectory"
> > +  echo "within the top-level corpus directory."
> > +  exit 1;
> > +fi
> > +
> > +wsj0=$1
> > +
> > +dir=`pwd`/data/local/data
>
> Should we change all parts (there are many, and actually many other
> recipes also use pwd often)?
> I actually don't understand the difference, and it would be great if you
> let me know the benefit of using $PWD.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/730/files/975abc99896ff7b0c5f6dced6109fc31cb40d34e#r60955587>
>